### DISCUSSION TITLE
Layer data rendered in wrong position until map is panned or zoomed

### DISCUSSION THREAD

**Discussion by jeffmacinnes** [visgl/deck.gl#5505] [Q&A] [answered]
Hi there! Hoping to get some insight/advice on an issue with layer data rendering in the wrong locations on the base map (initially, at least). When the page first loads, the layer is rendered in the wrong location. Once I interact with the map (via panning or zooming), the layers update to the correct location. Here's a minimal example where I'm just showing a single arc from the US to Australia

```javascript
import React from 'react';
import DeckGL from '@deck.gl/react';
import { ArcLayer } from '@deck.gl/layers';
import { StaticMap } from 'react-map-gl';

// --- Map settings
const INITIAL_VIEW_STATE = {
  longitude: 15.98,
  latitude: 44.9,
  zoom: 0,
  pitch: 20
};

const data = [
  {
    src: {
      lng: -99.94, // US
      lat: 40.43
    },
    dst: {
      lng: 133.75, // Australia
      lat: -25.27
    }
  }
];

const TestMap = (props) => {
  const layers = [
    new ArcLayer({
      data,
      id: 'arc-layer',
      getWidth: 4,
      getSourcePosition: (d) => [d.src.lng, d.src.lat],
      getTargetPosition: (d) => [d.dst.lng, d.dst.lat],
      getSourceColor: [255, 0, 0, 50],
      getTargetColor: [255, 0, 0, 50]
    })
  ];

  return (
    <DeckGL
      initialViewState={INITIAL_VIEW_STATE}
      controller={true}
      layers={layers}
    >
      <StaticMap reuseMaps mapboxApiAccessToken={process.env.MAPBOX_TOKEN} />
    </DeckGL>
  );
};

export default TestMap
```

When the site first loads, the arc is positioned as 
<img width="714" alt="Screen Shot 2021-02-18 at 8 43 56 PM" src="https://user-images.githubusercontent.com/14912413/108459471-95f5b480-722b-11eb-8ea4-ec397e28974b.png">

Only once I zoom or pan the map does it update to the correct location:
<img width="708" alt="Screen Shot 2021-02-18 at 8 44 18 PM" src="https://user-images.githubusercontent.com/14912413/108459552-b9b8fa80-722b-11eb-91d2-2ea7b4f8d775.png">

The same problem occurs with other layer types (tested with `ColumnLayer` as well) and across multiple browsers (Chrome, Firefox, Safari). And it's independent of the `StaticMap` component (commenting the `StaticMap` out produces the same behavior, just no underlying map). 

Going in circles trying to figure out what I'm missing. Embarrassed to admit I even tried programmatically simulating mouse events to get the map to update, but no luck 🙃. Any help would be immensely appreciated, thanks!

Using `deck.gl@8.4.6`

**Comment by Pessimistress** [comment, marked as answer]
This is discussed in https://github.com/visgl/deck.gl/issues/4965

**Reply by jeffmacinnes** [reply]
Perfect! Setting the latitude to `0` fixed it. Many thanks 🙏!
